static/src/stylesheets/layout/_ab-new-header-test-variant.scss

```diff
@@ -241,6 +204,93 @@ $gutter-medium: 37px;
     padding-right: 3px;
 }
 
+/*****************
+ * Edition Picker
+ *****************/
+
+.edition-picker {
+    @include fs-textSans(3);
+    order: -1;
+    color: $news-support-1;
+    // Override from fs-textSans mixin
+    line-height: 1;
+    padding: ($gs-baseline) ($gs-gutter / 2);
+
+    @include mq(mobileLandscape) {
+        margin-left: $gs-gutter / 2;
+    }
+
+    &:hover {
+        outline: none;
+        color: #ffffff;
+    }
+}
+
+.inline-world__svg {
+    position: absolute;
+    left: 2px;
+    top: 5px;
+    fill: currentColor;
+    min-width: 1em;
+    min-height: 1em;
+
+    &:hover {
+        fill: #ffffff;
+    }
+}
+
+.edition-picker__checkbox {
+    display: none;
+
+    &:checked {
```

Comment:
Changes we discussed have been made. Thanks for those suggestions @regiskuckaertz